src/components/BottomBarTabView.vue

```diff
@@ -30,11 +30,13 @@
           v-if="!seeMoreOpen"
           class="bottombarSeeMoreTab-arrow"
           src="@/assets/images/uparrow-white.svg"
+          alt="up arrow"
```

Comment:
I think it should see something like "collapse see more" instead of just repeating the name of the icon file.